server/src/models/Movie.js

```diff
@@ -14,6 +14,33 @@ class Movie extends Model {
       },
     };
   }
+
+  static get relationMappings() {
+    const { User, Review } = require("./index.js");
+
+    return {
+      users: {
```

Comment:
this association is not necessary. While the ERD might look like the association is a many-to-many, it's not! We want to track that a Movie has many Reviews (that association looks great). When it comes to Users, we are more interested in the one User who created a given review. This association set up for `users` is saying "show me all of the users who have left a review for this movie", and we probably don't have a use case for that. A good rule of thumb is to only add associations in relationMappings that you are _actually using_. This is contrary to what we have done the last few weeks having you set up all possible associations for practice. Going forward we only need to define the ones we will use in the app!